sign/eddilithium3/eddilithium_test.go

```diff
@@ -0,0 +1,106 @@
+package eddilithium3
+
+import (
+	"encoding/binary"
+	"testing"
+)
+
+func BenchmarkVerify(b *testing.B) {
+	// Note that the expansion of the matrix A is done at Unpacking/Keygen
+	// instead of at the moment of verification (as in the reference
+	// implementation.)
+	var seed [32]byte
+	var msg [8]byte
+	var sig [SignatureSize]byte
+	pk, sk := NewKeyFromSeed(&seed)
+	SignTo(sk, msg[:], sig[:])
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		// We should generate a new signature for every verify attempt,
+		// as this influences the time a little bit.  This difference, however,
+		// is small and generating a new signature in between creates a lot
+		// pressure on the allocator which makes an accurate measurement hard.
+		Verify(pk, msg[:], sig[:])
+	}
+}
+
+func BenchmarkSign(b *testing.B) {
+	// Note that the expansion of the matrix A is done at Unpacking/Keygen
```

Comment:
Let's hide the details of the implementation of dilithium.

Reply:
I adjusted the note.